filebeat/processor/add_kubernetes_metadata/matchers.go

```diff
@@ -78,59 +78,79 @@ func newLogsPathMatcher(cfg common.Config) (add_kubernetes_metadata.Matcher, err
 // Docker container ID is a 64-character-long hexadecimal string
 const containerIdLen = 64
 
-// Pod UID is on the 5th index of the path directories
-const podUIDPos = 5
-
 func (f *LogPathMatcher) MetadataIndex(event common.MapStr) string {
 	value, err := event.GetValue("log.file.path")
-	if err == nil {
-		source := value.(string)
-		f.logger.Debugf("Incoming log.file.path value: %s", source)
+	if err != nil {
+		f.logger.Errorf("Error extracting log.file.path from the event")
```

Comment:
We may still need to log `source` here so as to have a more clear view of how the event that is related to this error looks like.

Reply:
do you mean `event` instead of `source`? this error might only occur in case `event` map does not contain `log.file.path`(=`source`) key

Reply:
Yeap my wrong, `event` is what I was thinking about. Logging the whole `event` would be helpful.

Reply:
Also maybe just a debug message would be enough, since ERRORs would flood a basic case when the processor for some reason does not work.

Reply:
fixed: https://github.com/elastic/beats/pull/28868/commits/e8ab1fed7e7d474f0601a3145f5799128a581f76